tensorrt_llm/_torch/attention_backend/sparse/dsa.py

```
@@ -597,13 +597,15 @@ def update_spec_dec_param(
         model_is_wrapped: bool = False,
         spec_metadata: Optional['SpecMetadata'] = None,
         spec_tree_manager: Optional['SpecTreeManager'] = None,
+        spec_decoding_tensor: Optional['SpecDecodingTensor'] = None,
```

Flake8 (7.3.0)

[error] undefined name 'SpecDecodingTensor'

(F821)